src/hints/HintSequencePlayer.js

```diff
@@ -0,0 +1,71 @@
+import { IHintPlayer } from './IHintPlayer';
+import { Debugger } from '../debug';
+
+/**
+ *
+ * @export
+ * @class HintSequencePlayer
+ */
+export class HintSequencePlayer extends IHintPlayer {
+  /**
+   * Creates an instance of HintSequencePlayer.
+   * @memberof HintSequencePlayer
+   */
+  constructor() {
+    super();
+    /** @type {function()[]} */
```

Comment:
What's this doing here?

Reply:
mostly for auto complete. it's not strictly necessary.